src/components/CreateAccountForm/CreateAccountForm.js

```diff
@@ -227,29 +221,6 @@ export default class CreateAccountForm extends Component {
               onChange={(isValid) => { }}
             />
           </div>
-          <div className='form-group'>
```

Comment:
I spent a lot of time on this. Sad to see it go :( lol but it had to happen. Thank you!